B2MML-ProcessEvents.xsd

```diff
@@ -2062,34 +2060,33 @@
 
   <xsd:complexType name="WorkReconciledEventType">
     <xsd:sequence>
-      <xsd:element name="Notify" type="TransNotifyType" />

```

Comment:
Issue #5  this line is not required as it is already on line 2047, and brings it in to line with all other Notify structures.